.kokoro

```diff
@@ -174,6 +174,35 @@ run_cocoapods() {
   bash <(curl -s https://codecov.io/bash)
 }
 
+# For local runs, you must set the following environment variables:
+#
+#   DANGER_GITHUB_API_TOKEN -> Create a token here: https://github.com/settings/tokens.
+#                              Must have public_repo scope.
+#   DANGER_TEST_PR="###"    -> The PR # you want to test danger against.
+#
+# And you'll likely have to install danger:
+#
+#   yarn add danger
+#
+run_danger() {
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    # Move into our cloned repo
+    cd github/repo
+
+    export DANGER_TEST_PR="$KOKORO_GITHUB_PULL_REQUEST_NUMBER"
+
+    # Install danger
+    yarn add danger
+  fi
+
+  # We're not a supported CI, so we have to pretend to be one.
+  export DANGER_FAKE_CI="YEP"
```

Comment:
Is "YEP" a documented value for this variable in Danger?

Reply:
```
You can manually trigger danger against a pull request on the command line by setting the following environmental variables:

export DANGER_FAKE_CI="YEP"
export DANGER_TEST_REPO='username/reponame'
```